packages/svelte/src/internal/client/runtime.js

```diff
@@ -260,7 +331,7 @@ export function execute_reaction_fn(signal) {
 	current_untracking = false;
 
 	try {
-		let res = signal.fn();
+		let res = (0, signal.fn)();
```

Comment:
pushed this change — it gets rid of the `[as fn]`:

<img width="361" alt="image" src="https://github.com/sveltejs/svelte/assets/1162160/4bed0ca5-8f29-48b8-90b1-65acc6feb417">

<img width="303" alt="image" src="https://github.com/sveltejs/svelte/assets/1162160/e122d3cc-f9be-4d3b-b2ec-47c7da8491c0">

Reply:
Nice!